schema.graphql

```diff
@@ -67,3 +48,35 @@ type UniquesTransfer @entity {
   timestamp: Float @index
   success: Boolean
 }
+
+type Pot @entity {
+  id: ID!
+  potId: Float! @index 
+  owner: String! @index
+  feeQuota: BigInt!
+  reserveQuota: BigInt!
+  sponsorshipType: String!
+  sponsoredAccounts: [PotBalance] @derivedFrom(field: "pot")
+  createdAt: Float @index
+  updatedAt: Float @index
+
+}
+
+type PotBalance @entity @compositeIndexes(fields: [["pot", "account"]]) {
+  id: ID!
+  pot: Pot!
+  account: Account!
+  feeQuota: BigInt!
+  reserveQuota: BigInt!
+  createdAt: Float @index
+  updatedAt: Float @index
+}
+
+type Account @entity {
+  id: ID!
+  address: String! @index
+  potBalance: [PotBalance] @derivedFrom(field: "account")
```

Comment:
User balance is not the same as the pot balance.

Reply:
How about you remove this field to simply the relationship?`PotBalance` is already offering composite indexing for pot and account.

Reply:
Maybe the name is not the best `PotBalance` => `AccountPotBalance`
This entity simplify the queries of data representation, otherwise we need complex queries and a long table